### PR TITLE
Add header about Infinity deprecation redirecting to other solutions

### PR DESCRIPTION
**Pull request by fxmarty**
Following: https://github.com/huggingface/blog/issues/675

### CODE REVIEW

**Review by julien-c — CHANGES_REQUESTED** [review]

File: infinity-cpu-performance.md [hunk 1]
```diff
@@ -2,6 +2,30 @@
...
+  padding: 10px;
+  border: 2px solid green;
+  margin-left: auto;
+  margin-right: auto;
+  width: 60em;
+  text-align: left;
+}
+</style>
+
+<div class="infobox">
```
[the remark this answers is not in the document]
Comment (julien-c) [reply infinity-cpu-performance.md:19]: yes i don't think it's ideal

File: infinity-cpu-performance.md [hunk 2]
```diff
@@ -2,6 +2,30 @@
...
+  Infinity was a research product and is no longer maintained by 🤗 Hugging Face. The following solutions are available:
+
+  * For easy and hosted inference, we recommend using [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index).
+
+  * To accelerate models on CPU, we recommend using [🤗 Optimum Intel](https://huggingface.co/blog/openvino) integration or [🤗 Optimum ONNX Runtime](https://huggingface.co/docs/optimum/main/en/onnxruntime/overview) integration.
+
+  * Our team can assist you in the optimizations for training and inference! Reach out to us through the [Expert Acceleration Program](https://huggingface.co/support).
```
Comment (jeffboudier) [inline infinity-cpu-performance.md:26]:
```suggestion
  Infinity is no longer offered by Hugging Face as a commercial inference solution. To deploy and accelerate your models, we recommend the following new solutions:

  * [Inference Endpoints](https://huggingface.co/docs/inference-endpoints/index) to easily deploy models on dedicated infrastructure managed by Hugging Face

  * Our open source acceleration libraries [🤗 Optimum Intel](https://huggingface.co/blog/openvino) and [🤗 Optimum ONNX Runtime](https://huggingface.co/docs/optimum/main/en/onnxruntime/overview) to accelerate your models on CPU.

  * Hugging Face [Expert Acceleration Program](https://huggingface.co/support), a commercial service for Hugging Face experts to work directly with your team to accelerate your Machine Learning roadmap and models. 
```

**Review by jeffboudier — COMMENTED** [review]
Thanks for starting this PR! Suggested some copy.

File: infinity-cpu-performance.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fxmarty) [reply infinity-cpu-performance.md]: done

**Comment by fxmarty** [comment]
Thanks!

**Review by julien-c — COMMENTED** [review]
can you share a screenshot of what it looks like, if you have one?

Thanks!

**Comment by fxmarty** [comment]
I'm not sure if it will work since there is no preview available for the blog, but:

image


image

**Comment by fxmarty** [comment]
@julien-c @jeffboudier How does it sound?

**Comment by philschmid** [comment]
@fxmarty you should be able to run the blog locally using codespaces.

**Comment by fxmarty** [comment]
Awesome thanks for the recommentation, it works!

Here's how it looks like (did not manage to have the bullets in black):

image

File: infinity-cpu-performance.md [hunk 3]
```diff
@@ -38,9 +36,19 @@ thumbnail: /blog/assets/46_infinity_cpu_performance/thumbnail.png
     </a>
 </div>
 
-
 <script async defer src="https://unpkg.com/medium-zoom-element@0/dist/medium-zoom-element.min.js"></script>
 
+<br>
+<div style="background-color: #e6f9e6; padding: 8px; outline: 2px solid; border-radius: 10px; text-align: left;">
+  Infinity is no longer offered by Hugging Face as a commercial inference solution. To deploy and accelerate your models, we recommend the following new solutions:
```
Comment (jeffboudier) [inline infinity-cpu-performance.md:43]:
```suggestion
  December 2022 Update: Infinity is no longer offered by Hugging Face as a commercial inference solution. To deploy and accelerate your models, we recommend the following new solutions:
```

**Review by jeffboudier — CHANGES_REQUESTED** [review]
lgtm! Thanks for the screenshot. Made a suggestion to contextualize the inserted block.

**Comment by fxmarty** [comment]
@jeffboudier can you approve?

**Comment by philschmid** [comment]
Maybe @stevhliu can also take a look regarding the language and @Reeray if he has some UI improvements, i think there is room to improve to better fit the Hub design.

**Review by stevhliu — COMMENTED** [review]
Looks good to me!

**Comment by fxmarty** [comment]
Let me know if there is anything I should do.

**Review by julien-c — APPROVED** [review]
lgtm

**Pull request merged**